iceoryx_examples/icedelivery_in_c/README.md

```diff
@@ -34,7 +34,7 @@ Let's take a look at the `receiving` function that comes with the
 
     ```c
     const char APP_NAME[] = "iox-c-subscriber";
-    iox_runtime_init("APP_NAME");
+    iox_runtime_init(APP_NAME);
```

Comment:
Hahaha, is this correct in the example itself? Another reason to use geoffrey ;)

Reply:
I checked it and the example itself was correct.